orca-pipelinetemplate/src/main/java/com/netflix/spinnaker/orca/pipelinetemplate/tasks/v2/DeleteV2PipelineTemplateTask.java

```diff
@@ -50,7 +50,7 @@ public TaskResult execute(Stage stage) {
     String tag = (String) stage.getContext().get("tag");
     String digest = (String) stage.getContext().get("digest");
 
-    Response _ = front50Service.deleteV2PipelineTemplate(templateId, tag, digest);
+    front50Service.deleteV2PipelineTemplate(templateId, tag, digest);
```

Comment:
Java can't use `_` as an identifier (not sure why this was working previously as we haven't changed Java version). However, this variable was totally pointless anyway.